esmvaltool/config-developer.yml

```diff
@@ -21,7 +21,7 @@
 CMIP6:
   input_dir:
     default: '/'
-    BADC: '[institute]/[dataset]/[exp]/[ensemble]/[mip]/[short_name]/[grid]/[latestversion]'
+    BADC: '[activity]/[institute]/[dataset]/[exp]/[ensemble]/[mip]/[short_name]/[grid]/[latestversion]'
```

Comment:
wait - is `activity` coded up in `_data_finder.py` as a valid tag?

Reply:
All tags are valid for data_finder as long as you define it for the dataset (or we add them automatically because we can deduce them)

@sloosvel is working on a pull request that will be able to deduce it from the experiment, at least for some cases